client/src/components/PickupDetail.js

```diff
@@ -44,10 +50,10 @@ const PickupDetail: () => React$Node = () => {
         </div>
       </div>
       <div class="container-fluid m-1">
-        <a role="button" href="/add-pickup" className="btn btn-primary mx-1">
+        <a role="button" onClick={backToLanding} className="btn btn-primary mx-1">
```

Comment:
what's the advantage of using history than `href`? seems to me `href` is simpler.

Reply:
From my understanding, by history, everytime when we direct to a page we already loaded once, our browser does not need to reload it again, as the page was just hidden somewhere for further use.